lib/health_cards/health_card_constraints.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'base64'
+require 'zlib'
+require 'json/minify'
+
+module HealthCards
+  ## Functionality for "Health Cards are Small" section of the Smart Health Cards Specification
+  module HealthCardConstraints
+    def bundle_operations(payload)
```

Comment:
This method needs a better name. Method names should usually be verbs. `bundle_operations` doesn't give enough insight into what the method does.

Reply:
Switched name to `strip_fhir_bundle()`